lib/manager/cloudbuild/index.ts

```diff
@@ -0,0 +1,7 @@
+import { extractPackageFile } from './extract';
+
+export { extractPackageFile };
+
+export const defaultConfig = {
+  fileMatch: [/cloudbuild\.y[a]?ml$/],
```

Comment:
Should be a string

Reply:
Thanks! Is there a standard on how to handle .yml vs .yaml with that approach?

Reply:
Seemed like `fileMatch: ['(^|/)cloudbuild.ya?ml']` might work as intended

Reply:
Yes, that would.